### ISSUE TITLE
[BUG] CTD on approach

### ISSUE THREAD

**Issue by Kebushay** [saltysimulations/salty-747#195]
**Version of the mod**
0.3

**Describe the bug**
Hey I'm getting an issue where I set up an approach in the MCDU but as i start to get close to it, it just dissappears from the MCDU and the plane starts flying in a circle. When I try to set up a new approach, nothing changes and eventually CTD. I noticed the MCDU and plane behaviour is incorrect but I can see on the map in the plane and in the VFR map that the waypoints are still there.

**To Reproduce**
1. Load plan from simbrief
2. Fly plane as per normal
3. Follow ATC orders and flight plan
4. Issue occurs as you reach approach waypoints

**Expected behavior**
For plane to fly approach

**Additional information**
Have not tested

**Comment by ninjomcs** [comment]
is this something you could reproduce multiple times?

**Comment by Kebushay** [comment]
Yay a response! Yes, it happened twice. Now that I've thought about it I should add as well, when loading the flightplan from simbrief, it doesn't load all the waypoints properly, like sometimes it'll only load one waypoint or two (to test whether it was a problem with simbrief interacting with MSFS, I loaded the same plan into A32NX mod with no issues). So I could tell when I tried for a third time it was going to happen again.
